docs/csharp/versioning.md

```diff
@@ -81,19 +84,20 @@ your app's `app.config` file to use the `1.0.1` patch version of `ReferencedLibr
 </dependentAssembly>
 ```
 
-Go [here](https://msdn.microsoft.com/en-us/library/7wd6ex19(v=vs.110).aspx) to get more information on redirecting assembly versions.
+> [!NOTE]
+> This approach will only work if the new version of `ReferencedLibrary` is binary compatible with your app.
+> See the `Backwards Compatibility` section above for changes to look out for when determining compatibility.
```

Comment:
Backtics shouldn't be used here, since it's not code. And since you're referencing a previous section, it would be nice to make it a link:

```md
> See the [Backwards Compatibility](#backwards-compatibility) section above …
```